virttest/qemu_devices/qdevice_format.py

```diff
@@ -132,6 +132,7 @@ def __init__(self):
                 "align": self._normalize_data_size,
                 "host-nodes": self._int_in_list,
                 "prealloc-threads": self._str_to_dec,
+                "attributes": self._hex_in_str_to_dec,
```

Comment:
Just a reminder, we should always set the attributes with a hex string in cfg even when the json setting is support in future.